tests/wpt/mozilla/meta/webgl/conformance-1.0.3/conformance/state/gl-object-get-calls.html.ini

```diff
@@ -1,29 +1,53 @@
 [gl-object-get-calls.html]
-  expected: ERROR
-  [WebGL test #33: gl.getRenderbufferParameter(gl.RENDERBUFFER, gl.RENDERBUFFER_WIDTH) should be 2. Threw exception TypeError: gl.getRenderbufferParameter is not a function]
+  expected: TIMEOUT
```

Comment:
Looking at the last [log](http://build.servo.org/builders/linux-rel-wpt/builds/8557/steps/shell__4/logs/filtered-wpt-errorsummary.log), I guess this should be `OK`, then? Not sure why the script generated this to be a `TIMEOUT` and what's happening with this test in general.

cc @nox to the rescue.

Reply:
IDK, I'm pushing again without the `TIMEOUT`

Reply:
Alright, let's see. Hope it works now :).